Modules/TRD/src/TrackingTask.cxx

```diff
@@ -0,0 +1,295 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   TrackingTask.cxx
+/// \author Salman Malik
+///
+
+#include "QualityControl/QcInfoLogger.h"
+#include "DataFormatsTRD/TrackTRD.h"
+#include "DataFormatsTRD/TrackTriggerRecord.h"
+#include "DataFormatsTRD/Constants.h"
+#include "DataFormatsTRD/Tracklet64.h"
+#include "DataFormatsGlobalTracking/RecoContainer.h"
+#include "DataFormatsTRD/CalibratedTracklet.h"
+#include "TRDQC/Tracking.h"
+#include "TRD/TrackingTask.h"
+#include <Framework/InputRecord.h>
+#include <Framework/InputRecordWalker.h>
+#include <Framework/DataRefUtils.h>
+#include <Framework/DataSpecUtils.h>
+#include "CCDB/BasicCCDBManager.h"
+#include "TLine.h"
+#include "TCanvas.h"
+#include "TStyle.h"
+#include "TH1.h"
+
+using namespace o2::trd;
+using namespace o2::trd::constants;
+
+namespace o2::quality_control_modules::trd
+{
+
+TrackingTask::~TrackingTask()
+{
+}
+
+void TrackingTask::retrieveCCDBSettings()
+{
+  if (auto param = mCustomParameters.find("ccdbtimestamp"); param != mCustomParameters.end()) {
+    mTimestamp = std::stol(mCustomParameters["ccdbtimestamp"]);
+    ILOG(Info, Support) << "configure() : using ccdbtimestamp = " << mTimestamp << ENDM;
+  } else {
+    mTimestamp = o2::ccdb::getCurrentTimestamp();
+    ILOG(Info, Support) << "configure() : using default timestam of now = " << mTimestamp << ENDM;
+  }
+}
+
+void TrackingTask::initialize(o2::framework::InitContext& /*ctx*/)
+{
+  ILOG(Info, Support) << "initialize TRD TrackingTask" << ENDM;
+
+  retrieveCCDBSettings();
+  buildHistograms();
+}
+
+void TrackingTask::startOfActivity(Activity& activity)
+{
+  ILOG(Info, Support) << "startOfActivity " << activity.mId << ENDM;
+}
+
+void TrackingTask::startOfCycle()
+{
+  ILOG(Info, Support) << "startOfCycle" << ENDM;
+}
+
+void TrackingTask::monitorData(o2::framework::ProcessingContext& ctx)
+{
+  ILOG(Info, Support) << "monitorData" << ENDM;
```

Comment:
Please, refrain from ever logging messages from `monitorData`. It will flood the infologger given the frequency at which this method is called. 
You could still do it if needed but using `Debug` and not `Info` and `Devs` instead of `Support`.